guide/using-the-ipr-robot.md

```diff
@@ -0,0 +1,73 @@
+## Using the IPR robot
+
+%figure "IPR"
+
+![ipr.png](images/ipr.png)
+
+%end
+
+The IPR robot is a 6-DOF robotic arm equipped with a gripper composed of 2 rotational fingers.
+It is developed by [Neuronics](http://www.neuronics.be/).
```

Comment:
This link is wrong! Not Neuronics was a Swiss company which doesn't exist any more, not a Belgian company... 😆